### PR TITLE
#34 Feature: routes CRUD operations for operator

### PR DESCRIPTION
**Pull request by Salim-54**
### What does this PR do?
- routes CRUD operations for operator
### Description of Task to be completed?

As an operator, I want to create routes so that buses can travel in pre-defined routes. As an operator, I want to create routes so that users can view in pre-defined routes.

- The operator should be able to create, delete, and update a route.
- The operator should be able to view a paginated list of routes in the system.
- The users should be able to view a route's details
- Skeleton UIs/screen are/is shown when the page loads/is loading.
- The page is responsive.
- The page is responsive.

### How should this be manually tested?
`Clone the repo`

`checkout` to branch `ft-operator-crud-for-routes-34`

pull the changes by `git pull`

install all packages by `running npm install`

start app by running `npm run start`

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[[Trello Board](https://trello.com/c/TgBkFOb5/34-crud-operations-for-routes)
Screenshot (302)
Screenshot (301)
Screenshot (303)

### CODE REVIEW

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good

**Review by tysepa — APPROVED** [review]

**Review by Xldivin — APPROVED** [review]

File: src/component/UpdateRouteModal.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            ) : null}
+
+              <label className="font-semibold">Destination</label>
+              <input
+                type="text"
+                name="destination"
+                id="destination"
+                data-testid="busPlate-input"
+                // onChange={handleChangeUpdate}
+                // defaultValue={`${destinationUpdate}`}
```
Comment (thepatrickniyo) [inline src/component/UpdateRouteModal.js]: Remove these comments for the best code quality

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
The work meets expectations just remove those commented source codes

**Review by Xldivin — APPROVED** [review]

**Comment by thepatrickniyo** [comment]
Finally rebase your branch on `develop` to remove the conflicts

**Review by Hodal66 — APPROVED** [review]

**Review by tysepa — APPROVED** [review]

**Review by Niyonkuru1 — APPROVED** [review]
Looks great!

**Review by thepatrickniyo — APPROVED** [review]

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**